doc/glossary.rst

```diff
@@ -76,13 +76,31 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward solution
+        The forward solution or forward model (abbr. ``fwd``) is a linear
+        operator capturing the relationship between each dipole location in the
+        :term:`source space` and the expected pattern of activity on the
+        sensors corresponding to that dipole (AKA, the "lead fields"). Forward
+        solutions are obtained by solving the Maxwell equations describing
```

Comment:
I'd rather not talk about 'propagation': the solutions are calculated under quasi-static assumptions, so specifically no E/B-coupling. How about simply:

```suggestion
        Calculating a forward solution requires that a model of the tissues of the head is defined, which encapsulates the geometry and electrical conductivity of the different compartments (see :term:`boundary ...`).
```